torch/fx/graph.py

```diff
@@ -533,12 +533,16 @@ def emit_node(node : Node):
                 meta_val = node.meta.get('val', node.meta.get('tensor_meta', None))
 
                 # use string as annotation, to make it valid python code
+                # TODO: sparse layout is shown here for debugging
                 if isinstance(meta_val, FakeTensor):
-                    maybe_type_annotation = f': "{dtype_abbrs[meta_val.dtype]}{stringify_shape(meta_val.shape)}"'
+                    if meta_val.sparse_layout is None:
```

Comment:
This if-block can be avoided if layout is included into `maybe_type_annotation`.